server.js

```diff
@@ -83,8 +83,10 @@ app.get('/releases', routes.releases.index)
 app.get('/spectron', routes.spectron)
 app.get('/userland', routes.userland.index)
 app.get('/userland/*', routes.userland.show)
-
 app.use('/crowdin', routes.languages.proxy)
+app.get('/search/:searchIn*?', routes.search.index)
```

Comment:
I'm unfamiliar with this route handle syntax. What does `:searchIn*?` mean?

Reply:
That's assigning whatever comes after `/search/` as `searchIn`, and passing it to the handler. So `/search/repo?query=xyz` would have `searchIn === 'repo'`.